src/Avalonia.FuncUI/Library.fs

```diff
@@ -33,7 +33,7 @@ module internal Extensions =
         member this.SubscribeWeakly(callback: 'a -> unit, target) =
             Observable.subscribeWeakly(this, callback, target)
 
-    type IInteractive with
+    type Interactive with
```

Comment:
Another change that effects this bit is that the Avalonia change at https://github.com/AvaloniaUI/Avalonia/pull/9749 has removed its usage of System.Reactive and replaced it with a bunch of (mostly internal) things.

I added a local reference to System.Reactive to make it build, but i'm not sure of the 'best' approach (still learing how all this hangs together) - e.g. Avalonia now seems to have an internal implementation of Observable.Create that uses a minimal local class, so I wondered if doing something like

```fsharp
{ new IObservable<'args>
  with member this.Subscribe(observer: IObserver<'args>) = sub.Invoke(observer) }
```

would work instead of importing Reactive just for Observable.Create?

There are also 3 references to Observable.Subscribe that require Reactive, but the comment at https://github.com/AvaloniaUI/Avalonia/pull/9749#issuecomment-1378290972 suggests there might be more changes coming on the Avalonia side there

Reply:
should be good 👍

Reply:
The general bit about System.Reactive, or the local object expression suggestion?

Reply:
I'd prefer not having a reference to 'System.Reactive'. So the object expression seems good to me.

Reply:
Would there be any value in breaking bits like this into separate PRs (in cases where they can be done without updating the Avalonia version)?

Reply:
The two cases where breaking this PR into smaller bits might make sense:
1) if the change could be released as a new preview version that would provide immediate value for end users.
2) if the change is big enough (touches enough files) that it will become harder to merge the longer it sits.

Reply:
The question was mainly that you don't have to do it all in one go (as you could leave the FuncUI code using a local System.Reactive reference to begin with and fix it later, *or* you could try removing local dependencies on System.Reactive without updating Avalonia, e.g. #261 ).
The value is probably more that it'd be more self contained to test.